.gitignore

```diff
@@ -4,3 +4,7 @@
 /*.xcodeproj
 xcuserdata/
 .swiftpm
+node_modules/
+.parcel-cache
+.vscode
```

Comment:
Since we'll now have cross-cutting concerns, it might be worth taking a moment to tailor a `.gitignore` file from a few of the relevant [GitHub templates](https://github.com/github/gitignore).

Reply:
I took a look and they are pretty exhaustive. Shall we just grab the Node one wholesale?